src/presentation/pages/Login.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Navbar from '../components/Navbar'
+import { Button, Divider, Input } from '@nextui-org/react'
+import { Link } from 'react-router-dom'
+import LinkIcon from '../assets/LinkIcon'
+
+export default function Login() {
+  return (
+    <>
+    {/* FIX: acertar largura dos componentes na página e suas responsividades */}
+    {/* FEATURE: conectar ao banco e fazer checagens necessárias p/ funcionamento do botão de login*/}
+    {/* FIX: remover navbar? */}
+      <Navbar />
```

Comment:
Sim, remover Navbar e adicionar a logo
```suggestion
{/* TODO: Adicionar logo */}
```